### PR TITLE
All tests have run successfully

### PR DESCRIPTION
**Pull request by qzzhang**
This is what was printed out on the terminal after all runs ended:
-------------------------------------------------------------------------------------------
All tests successful.
Files=8, Tests=126, 10063 wallclock secs (19.87 usr  1.26 sys + 729.60 cusr 104.45 csys = 855.18 CPU)
Result: PASS
-------------------------------------------------------------------------------------------

Modifications I have made are mostly to streamline with changes of file names and file locations in the last merged PR.

### CODE REVIEW

File: lib/RAST_SDK/RAST_SDKImpl.pm [hunk 1]
```diff
@@ -4,8 +4,8 @@ use Bio::KBase::Exceptions;
 # Use Semantic Versioning (2.0.0-rc.1)
 # http://semver.org
 our $VERSION = '0.1.6';
```
Comment (ialarmedalien) [inline lib/RAST_SDK/RAST_SDKImpl.pm]: this needs to be updated

File: test/annotation_utils.t [hunk 2]
```diff
@@ -1546,37 +1530,32 @@ subtest '_write_gff_from_genome' => sub {
...
         "output_workspace" => $ws_name
     };
-
-    throws_ok {
+    lives_ok {
         $rast_ret = $annoutil->rast_genome($parms);
-    } qr/$allre/,
-        '$annoutil->rast_genome returns ERROR due to kmer data absence or other causes.';
+    } "annoutil->rast_genome call returns normally on genome $obj_Ecoli.";
+
```
Comment (ialarmedalien) [inline test/annotation_utils.t:1549]: Can you explain why these tests have gone from throwing errors to passing, without any change in the application logic?

File: test/annotation_utils.t [hunk 3]
```diff
@@ -1894,4 +1873,4 @@ subtest 'annoutil_uniq_functions' => sub {
 
 RASTTestUtils::clean_up();
 
-done_testing();
+done_testing(38);
```
Comment (ialarmedalien) [inline test/annotation_utils.t]: YMMV, but I wouldn't bother with putting in this test count -- it is a nuisance to update it when you add more tests to the file.

**Review by ialarmedalien — CHANGES_REQUESTED** [review]
The version numbers in the perl code need to be fixed.

File: lib/RAST_SDK/RAST_SDKImpl.pm (hunk 1 above)
Comment (qzzhang) [reply lib/RAST_SDK/RAST_SDKImpl.pm]: I agree, I just noticed my PR #82 was closed which has the change of version number.

File: test/annotation_utils.t (hunk 3 above)
Comment (qzzhang) [reply test/annotation_utils.t]: I thought it was a neat feature to track the test count.  You are right when adding or removing a test I would have to remember to update the count.  I will roll this back.

File: test/annotation_utils.t (hunk 2 above)
Comment (qzzhang) [reply test/annotation_utils.t:1549]: It was a missed update I should have made after I changed in the application logic earlier.  I made the change in one of the tests within the same subtest function, yet forgot to change the other two.

**Review by ialarmedalien — APPROVED** [review]

**Pull request merged**